apps/i3wm/i3wm.talon

```diff
@@ -2,35 +2,40 @@
 os: linux
 tag: user.i3wm
 -
-port <number_small>: user.system_command("i3-msg workspace number {number_small}")
-port ten: user.system_command("i3-msg workspace number 10")
-(port flip|flipper): user.system_command("i3-msg workspace back_and_forth")
-port right: user.system_command("i3-msg workspace next")
-port left: user.system_command("i3-msg workspace prev")
+port <number_small>:
+    user.system_command("i3-msg workspace number {number_small}")
+port ten:
+    user.system_command("i3-msg workspace number 10")
+(port flip | flipper):
+    user.system_command("i3-msg workspace back_and_forth")
+port right:
+    user.system_command("i3-msg workspace next")
+port left:
+    user.system_command("i3-msg workspace prev")
 
-(win|window) left: user.system_command("i3-msg focus left")
-(win|window) right: user.system_command("i3-msg focus right")
-(win|window) up: user.system_command("i3-msg focus up")
-(win|window) down: user.system_command("i3-msg focus down")
-((win|window) kill|murder): user.system_command("i3-msg kill")
-(win|window) stacking: user.system_command("i3-msg layout stacking")
-(win|window) default: user.system_command("i3-msg layout toggle split")
-(win|window) tabbed: user.system_command("i3-msg layout tabbed")
+(win | window) left: user.system_command("i3-msg focus left")
+(win | window) right: user.system_command("i3-msg focus right")
+(win | window) up: user.system_command("i3-msg focus up")
+(win | window) down: user.system_command("i3-msg focus down")
+((win | window) kill | murder): user.system_command("i3-msg kill")
+(win | window) stacking: user.system_command("i3-msg layout stacking")
+(win | window) default: user.system_command("i3-msg layout toggle split")
+(win | window) tabbed: user.system_command("i3-msg layout tabbed")
 
 reload i three config: user.system_command("i3-msg reload")
 restart i three: user.system_command("i3-msg restart")
 
-(full screen|scuba): user.system_command("i3-msg fullscreen")
+(full screen | scuba): user.system_command("i3-msg fullscreen")
 toggle floating: user.system_command("i3-msg floating toggle")
 focus floating: user.system_command("i3-msg focus mode_toggle")
 center window: user.system_command("i3-msg move position center")
-resize mode: user.system_command('i3-msg mode "resize"')
+resize mode: user.system_command("i3-msg mode \"resize\"")
 focus parent: user.system_command("i3-msg focus parent")
 focus child: user.system_command("i3-msg focus child")
 
 # resize helpers
 grow window:
-    user.system_command('i3-msg mode "resize"')
+    user.system_command("i3-msg mode \"resize\"")
```

Comment:
Minor point, but some formatters will use single quote if the string contains a double quote but not single

Reply:
Let’s not have that be blocking for the merge, but open an issue on the talonfmt repository. Shouldn’t be too hard.

Reply:
Implemented this in HEAD, will release soon.

Reply:
Fixed, should be working now if you look at the diffs again.